yamale/util.py

```diff
@@ -86,3 +95,39 @@ def get_iter(iterable):
         return iterable.items()
     else:
         return enumerate(iterable)
+
+
+def get_subclasses(cls, _subclasses_yielded=set()):
```

Comment:
Mutable default arguments in Python: http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments